apsuite/optics_analysis/tune_correction.py

```diff
@@ -0,0 +1,244 @@
+"""."""
+
+from collections import namedtuple as _namedtuple
+from copy import deepcopy as _dcopy
+import numpy as np
+from pymodels import bo, si
+import pyaccel
+
+__KNOBTYPES = _namedtuple('KnobTypes', ['QFs', 'QDs'])
+
+
+class KnobTypes(__KNOBTYPES):
+    """."""
+
+    @property
+    def ALL(self):
+        """."""
+        return self.QFs + self.QDs
+
+
+class TuneCorr():
+    """."""
+
+    SIDEFKNOBS = KnobTypes(
```

Comment:
I think that using underscore separators, like `SI_DEF_KNOBS`, makes code much more readable. And with code completion you don't even have to type awkward `_` char... don't you think?

Reply:
ok @xresende. I'll change it.

Reply:
done!